build_config.yml

```diff
@@ -11,4 +11,4 @@ git_repo_versions:
     h_client_build: master
   discover: master
   liquid_core: master
-  setup: master
+  setup: dont-use-ansible-deprecated-features
```

Comment:
I know you plan to change this, but can we keep it out of the git history altogether? It will make digging through commit logs harder.

Reply:
I'll rebase -i and remove that commit before merging.